docs/develop/itp-and-third-party-cookies.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Develop your Office Add-in to work with ITP when using third-party cookies
+description: How to work with ITP and Office Add-ins when using third-party cookies
+ms.date: 03/12/2021
+localization_priority: Normal
+---
+
+# Develop your Office Add-in to work with ITP when using third-party cookies
+
+If your Office Add-in requires third-party cookies, those cookies are blocked if Intelligent Tracking Prevention (ITP) is used by the browser runtime that loaded your add-in. You may be using third-party cookies to authenticate users, or for other scenarios, such as storing settings.
+
+If your Office Add-in and website must rely on third-party cookies, use the following steps to work with ITP.
+
+1. Set up [OAuth 2.0 Authorization](https://tools.ietf.org/html/rfc6749) with which the authenticating domain (in your case, the third-party that expects cookies) forwards an authorization token to your website which you consume and use to establish a first-party login session with a server-set Secure and [HttpOnly cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies).
+2. Use the [Storage Access API](https://webkit.org/blog/8124/introducing-storage-access-api/) with which the third-party can request permission to get access to its first-party cookies.
```

Comment:
```suggestion
2. Use the [Storage Access API](https://webkit.org/blog/8124/introducing-storage-access-api/) so that the third-party can request permission to get access to its first-party cookies.
```